src/main/java/com/elasticpath/tools/mavenminimal/config/XmlMvnMinConfig.java

```diff
@@ -148,7 +148,7 @@ public static XmlMvnMinConfig load() {
 		try {
 			JAXBContext jaxbContext = JAXBContext.newInstance(MvnMinConfigFile.class);
 			Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
-			MvnMinConfigFile configFile = (MvnMinConfigFile) jaxbUnmarshaller.unmarshal(new File(MVNMIN_CONFIG_FILE_NAME));
+			MvnMinConfigFile configFile = (MvnMinConfigFile) jaxbUnmarshaller.unmarshal(mvnminConfigFile);
```

Comment:
Are you sure this works? I would expect this to try to unmarshal the "mvnmin.xml" string, rather than reading the file...

Reply:
I basically refactored this.  A little above we have:
```
		File mvnminConfigFile = new File(MVNMIN_CONFIG_FILE_NAME);
```
so I just used that in place of what was already there.  And this seems legit: https://docs.oracle.com/javase/7/docs/api/javax/xml/bind/Unmarshaller.html#unmarshal(java.io.File)

Reply:
Ah, I missed that because the variable was already there (not part of the diff). Looks good!